content/ar/docs/setup/best-practices/certificates.md

```diff
@@ -0,0 +1,276 @@
+شهادات `front-proxy`  مطلوبة فقط إذا كنت تشغل kube-proxy  من أجل دعم
+[an extension API server](/docs/tasks/extend-kubernetes/setup-extension-api-server/).
+أيضا etcd يطبق TLS المتبادل لمصادقة العملاء و الأقران
+{{< /note >}}
+
+
+## أين يتم تخزين الشهادات
+
+إذا قمت بتثبيت كوبرنيتيز باستخدام kubeadm، فإن معظم الشهادات مخزنة في `/etc/kubernetes/pki`.
+جميع المسارات في هذه الوثائق نسبية إلى هذا الدليل، باستثناء شهادات حساب المستخدم التي يضعها kubeadm في `/etc/kubernetes`
+.
+
+
+## ضبط الشهادات يدويا
+
+إذا لم ترغب في أن ينشئ kubeadm الشهادات المطلوبة، فيمكنك إنشاؤها باستخدام شهادة جذر واحدة أو عن طريق توفير جميع الشهادات. راجع [الشهادات](/docs/tasks/administer-cluster/certificates/)  من أجل المزيد من المعلومات حول إنشاء سلطة شهادات خاصة بك .
+راجع [تسيير الشهادات عن طريق kubeadm](/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/)
+من أجل المزيد حول تسيير الشهادات.
+
+
+
+### شهادة الجذر الواحدة
+
+يمكنك إنشاء شهادة جذر واحدة، يتحكم فيها المسؤول. ثم يمكن لهذه الشهادة إنشاء عدة شهادات وسيطة، وتفويض جميع عمليات الإنشاء التالية إلى كوبرنيتيز نفسه.
+
+الشهادات المطلوبة:
+
+
+
+|         المسار                |          الاسم المشترك الافتراضي    |                  الوصف                                                                 |  
+|--------------------------------|-----------------------------------|---------------------------------------------------------------------------------------|
+| ca.crt,key                     | kubernetes-ca                     | Kubernetes general CA                                                                 |
+| etcd/ca.crt,key                | etcd-ca                           | For all etcd-related functions                                                        |
+| front-proxy-ca.crt,key         | kubernetes-front-proxy-ca         | For the [front-end proxy](/docs/tasks/extend-kubernetes/configure-aggregation-layer/) |
+
+
+بالإضافة إلى شهادات الاعتماد (CA) المذكورة أعلاه، من الضروري أيضًا الحصول على زوج مفتاح عام/خاص لإدارة حساب الخدمة، `sa.key` و `sa.pub`.
+المثال التالي يوضح ملفات المفاتيح والشهادات CA المبينة في الجدول السابق:
+
+
+```
+/etc/kubernetes/pki/ca.crt
+/etc/kubernetes/pki/ca.key
+/etc/kubernetes/pki/etcd/ca.crt
+/etc/kubernetes/pki/etcd/ca.key
+/etc/kubernetes/pki/front-proxy-ca.crt
+/etc/kubernetes/pki/front-proxy-ca.key
+```
+
+
+
+
+### كل الشهادات
+
+إذا لم ترغب في نسخ مفاتيح CA الخاصة إلى العنقود الخاصة بك، يمكنك إنشاء جميع الشهادات بنفسك.
+
+
+الشهادات المطلوبة:
+
+|  الاسم المميز الافتراضي     |    الشهادة الأم  |           إسم المالك (في الموضوع) |                    النوع  |                   المستضيفين (SAN)                      |
+|-------------------------------|---------------------------|-----------------------------------|------------------|-----------------------------------------------------|
+| kube-etcd                     | etcd-ca                   |                                   | خادم, مستخدم     | `<hostname>`, `<Host_IP>`, `localhost`, `127.0.0.1` |
```

Comment:
```suggestion
| kube-etcd                     | etcd-ca                   |                                   | خادم, مستخدم     | `<hostname>`, `<Host_IP>`, `localhost`, `127.0.0.1` |
```

this does not look very nice on the preview the `>` is malformed I do not know why

Reply:
Looks nice now but can you investigate why these elements did not inherit the `<div dir="ltr">` from the body tag

Reply:
I don't think we've got a layout that sets `dir="ltr"` yet (it's needed, and adding it should be a priority)

Reply:
Ah  by default it is RTL with arabic and we need to override it for this element